### PR TITLE
Add list of data sources for City of Los Angeles

### PR DESCRIPTION
**Pull request by vykster**
Opportunity data sets for Los Angeles. 

Let me know what I missed :heart:  cc/ @matikin9 @machikoyasuda @michaelkirk @traviskorte @thekaveman @kevinsmgov @LearningNerd @lauraerinmann @corwinstephen @correlator @chrislkeller @krainboltgreene @jericho1ne @ondrae @abhinemani @dickeyxxx @kurtdaradics @jyaistMap

### CODE REVIEW

File: _data/data-providers/la.yml [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+    description: "Listing of Active Businesses"
+    url: "https://data.lacity.org/A-Prosperous-City/Listing-of-Active-Businesses/6rrh-rzua"
+
+  - topic: "Crime"
+    description: "LA County Datasets, Criminal"
+    url: "https://data.lacounty.gov/browse?category=Criminal&limitTo=datasets&utf8=%E2%9C%93"
+
+  - topic: "Crime"
+    description: "Safe City Datasets"
+    url: "https://data.lacity.org/browse?category=A+Safe+City&limitTo=datasets&sortBy=relevance&utf8=%E2%9C%93&page=2"
```
Comment (michaelkirk) [inline _data/data-providers/la.yml]: why page=2? Could probably just be `https://data.lacity.org/browse?category=A+Safe+City&limitTo=datasets&sortBy=relevance`

Comment (corwinstephen) [inline _data/data-providers/la.yml]: @michaelkirk agreed

**Comment by machikoyasuda** [comment]
```- topic: "Street sweeping map and schedule"
description: "Bureau of Street Services"
url: http://bss.lacity.org/restricted_parking_sweeping_routes.cfm```

**Comment by vykster** [comment]
@machikoyasuda It's not machine readable :( Just pdf maps per Council District.

**Comment by machikoyasuda** [comment]
@vykster ah man! though i suppose machine-readable is debatable :) we need pdf-reading machines.

**Comment by vykster** [comment]
@rossdakin so how does this work?

**Pull request merged**